.github/dependabot.yml

```
@@ -1,12 +1,18 @@
-# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+# https://docs.github.com/en/code-security/dependabot/working-with-dependabot/dependabot-options-reference
 
```

```suggestion
---
```
Technically there should be a start-of-document indicator here, so we can add it while we're changing this file anyway.